### PR TITLE
Baryons as in van Daalen+ 2019

### PR DESCRIPTION
**Pull request by elisachisari**

### CODE REVIEW

File: pyccl/baryons/vandaalen19.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import numpy as np
+
+from .. import Pk2D
+from . import Baryons
+
+
+class vanDaalen19(Baryons):
+    """The "van Daalen+ 2019" model boost factor for baryons.
+
+    .. note:: First presented in van Daalen et al., arXiv:1906.00968.
```
Comment (damonge) [inline pyccl/baryons/vandaalen19.py:12]: Can we include a link? See e.g. https://github.com/LSSTDESC/CCL/blob/dd2aa5e0cc228505f85daa8d869f8706bd4d8e37/pyccl/baryons/schneider15.py#L13

File: pyccl/baryons/vandaalen19.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+              The boost factor is applied multiplicatively so that
+              :math:`P_{\\rm bar.}(k, a) = P_{\\rm DMO}(k, a)\\,
+              f_{\\rm vD19}(k, a)`.
+
+              Notice the model has only been tested at z=0 and is valid for
+              :math:`k\\leq 1 \\,h/{\\rm Mpc}.
+
+    Args:
+        fbar500c (:obj:`float`): the fraction of baryons in a halo within
```
Comment (damonge) [inline pyccl/baryons/vandaalen19.py:22]: Do you think we could allow for the 200c mass definition from their table 3 too? I just remember how annoying it is to have to translate between them if you can't choose to use 500c. E.g. you could rename this to simply `fbar` and then add another argument of the form `mass_def='500c'`, but which users could change to `'200c'`

File: pyccl/baryons/vandaalen19.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        an overdensity of 500 times the critical density, given in units
+        of the ratio of :math:`\\Omega_b` to :math:`\\Omega_m`. 
+        Default to 0.7 which is approximately compatible with observations.
+        See Figure 16 of the paper.
+
+    """
+    name = 'vanDaalen19'
+    __repr_attrs__ = __eq_attrs__ = ("fbar500c")
+
+    def __init__(self, fbar500c=0.7):
```
Comment (damonge) [inline pyccl/baryons/vandaalen19.py:32]: is 0.7 a reasonable default, or should we force people to choose a value?

**Review by damonge — COMMENTED** [review]
Awesome! Just a few first comments.

File: pyccl/baryons/vandaalen19.py (hunk 2 above)
Comment (elisachisari) [reply pyccl/baryons/vandaalen19.py]: Done

File: pyccl/baryons/vandaalen19.py (hunk 1 above)
Comment (elisachisari) [reply pyccl/baryons/vandaalen19.py]: Done and recompiled docs

File: pyccl/baryons/vandaalen19.py (hunk 3 above)
Comment (elisachisari) [reply pyccl/baryons/vandaalen19.py]: Yes, 0.7 is reasonable. See green band consistent with observations in their Fig 16.

**Review by damonge — COMMENTED** [review]
Looks good, thanks a lot!
Just one more comment, and flake8 is complaining.

File: pyccl/baryons/vandaalen19.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        # [k]=1/Mpc [k_use]=h/Mpc
+        a_use, k_use = a_use[:, None], k_use[None, :]/cosmo['h']
+
+        if(self.mass_def=='500c'):
+            avD19 = 2.215
+            bvD19 = 0.1276
+            cvD19 = 1.309
+            dvD19 = -5.99
+            evD19 = -0.5107
+        else:
```
[the remark this answers is not in the document]
Comment (elisachisari) [reply pyccl/baryons/vandaalen19.py:67]: Done, and documented

**Comment by damonge** [comment]
@elisachisari two final things
- Flake8 is complaining
- Can you add a unit test for these lines?
image

**Review by damonge — APPROVED** [review]
LGTM!

**Comment by elisachisari** [comment]
Comments addressed.

**Pull request merged**